### PR TITLE
[New Rule] Peripheral Device Discovery

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/445

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/discovery_peripheral_device.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/11/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of fsutil Windows utility to gather information about attached peripheral devices and components
```
Comment (peasead) [inline rules/windows/discovery_peripheral_device.toml:10]:
```suggestion
Identifies use of the fsutil Windows utility to gather information about attached peripheral devices and components
connected to a computer system.
```

**Review by peasead — APPROVED** [review]
LGTM

File: rules/windows/discovery_peripheral_device.toml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the fsutil Windows utility to gather information about attached peripheral devices and components
+connected to a computer system.
+connected to a computer system.
```
Comment (brokensound77) [inline rules/windows/discovery_peripheral_device.toml:12]:
```suggestion
```

File: rules/windows/discovery_peripheral_device.toml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+risk_score = 21
+rule_id = "0c7ca5c2-728d-4ad9-b1c5-bbba83ecb1f4"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+(process.name : "fsutil.exe" or process.pe.original_file_name : "fsutil.exe") and process.args : "fsinfo" and process.args : "drives"
+'''
```
Comment (brokensound77) [inline rules/windows/discovery_peripheral_device.toml:27]: It is better to always leave original file name checks case sensitive (of course verifying the actual case)

```suggestion
query = '''
process where event.type in ("start", "process_started") and
  (process.name : "fsutil.exe" or process.pe.original_file_name == "fsutil.exe") and 
  process.args : "fsinfo" and process.args : "drives"
'''
```

File: rules/windows/discovery_peripheral_device.toml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/11/02"
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the fsutil Windows utility to gather information about attached peripheral devices and components
+connected to a computer system.
```
Comment (bm11100) [inline rules/windows/discovery_peripheral_device.toml:10]:
```suggestion
Identifies use of the Windows file system utility (fsutil.exe ) to gather information about attached peripheral devices and components
connected to a computer system.
```

Comment (bm11100) [reply rules/windows/discovery_peripheral_device.toml:10]: may need re lint

**Review by bm11100 — APPROVED** [review]

**Pull request merged**